core/build/default.go

```diff
@@ -0,0 +1,9 @@
+//go:build !dev
+// +build !dev
```

Comment:
Do we still need to do both for compatibility?
I thought we could drop the legacy format since our go.mod sets a recent version.
```suggestion
//go:build !dev
```

Reply:
we don't as long as we enforce compilation from source to a specific version, removed